search-resources-api/src/main/java/no/unit/nva/dynamodb/IndexDocumentGenerator.java

```diff
@@ -118,6 +123,13 @@ private static URI extractDoi(JsonNode record) {
                 .orElseThrow();
     }
 
+    private static URI extractPublisherId(JsonNode record) {
+        return Optional.ofNullable(record)
```

Comment:
It occurs to me that the id of publisher should be a URI, as it stands, it is a document-internal node, which is unproblematic, but I think we will revisit this in a later PR.